### PR TITLE
add fix for google play audiobook player

### PR DESCRIPTION
**Pull request by PhilippIRL**
This just fixes some font color issues

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5468,6 +5468,18 @@ img[src^="data:image/png;"]
 
 ================================
 
+play.google.com/books/listen
+
+INVERT
+.gb_ua
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5474]: Change it to `a[href*="about/products"]` this selector is known to change every x weeks.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5468,6 +5468,18 @@ img[src^="data:image/png;"]
...
+
+CSS
+.chapter-item:not(.iron-selected) .chapter-title {
+  color: var(--darkreader-neutral-text) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5478]: Use 4 spaces as ident.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (PhilippIRL) [reply src/config/dynamic-theme-fixes.config]: `a[href*="about/products"]` doesn't seem to work in this case. I used `a[href*="books/audiobooks"]` instead.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (PhilippIRL) [reply src/config/dynamic-theme-fixes.config]: okay, done

**Pull request merged**